tests/testcases/simple-redis-rust/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name    = "simple-redis"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = [ "cdylib" ]
+
+[dependencies]
+# Useful crate to handle errors.
+anyhow = "1"
+# Crate to simplify working with bytes.
+bytes = "1"
+# General-purpose crate with common HTTP types.
+http = "0.2"
+spin-sdk = { path = "../../../sdk/rust"}
+
+[workspace]
+
+# Metadata about this component.
+[package.metadata.component]
+name = "spinhelloworld"
```

Comment:
Use the app name?  We have a tendency to copy-paste this name everywhere and it causes woes for the Rust SDK (although this position is probably benign).